### DISCUSSION TITLE
export to pdf wrong 【Help】

### DISCUSSION THREAD

**Discussion by MoneySail** [slidevjs/slidev#144] [Q&A]
i try to run `npm i -D playwright-chromium` and `npm run export` to export to a PDF file, but it show like the photo blow, and i want to know how to fix it? 

<img width="1011" alt="Snipaste_2021-05-21_00-12-51" src="https://user-images.githubusercontent.com/17845399/119013222-5de84400-b9c9-11eb-8362-6bad09f94ea8.png">

**Comment by MoneySail** [comment]
i do 'npm install playwright' but unuseful

**Comment by antfu** [comment]
Maybe read this https://playwright.dev/docs/installation?
